.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+| Q                | A
+| ---------------- | ---
+| Request type     | bugfix/feature
+| Tested scenarios | Description of tested scenarios
+| Fixed issues     | #-prefixed issue number(s), if any
```

Comment:
Maybe better to show these questions like this:

### Description
<! Provide a description of the changes proposed in the pull request ->

### Fixed Issues (if relevant)
<!--- Provide a list of fixed issues in the format #<issue_number>, if relevant  -->
1. #<issue_number>: Issue title
2. ...

### Tested scenarios
<!--- Provide a Description of tested scenarios -->
1. ...
2. ...

Reply:
example see: https://github.com/magento/magento2/compare/2.2-develop...2.0